ubuild.py

```diff
@@ -0,0 +1,11 @@
+from uranium import current_build
+
+current_build.config.set_defaults({
+    "package_name": "intake-nested-yaml-catalog",
+    "module": "intake_nested_yaml_catalog",
+    "binaries": {"enabled": True},
```

Comment:
we don't need the binaries: enabled flag here. That's an internal Zillow construct used to get around some poorly packaged builds and a lack of support around wheels. Both of which can now be resolved, but we can't remove this flag internally until all package have been updated.